dashboard/app/models/stage.rb

```diff
@@ -113,12 +113,7 @@ def summarize
         flex_category: localized_category,
         lockable: !!lockable,
         levels: cached_script_levels.reject(&:bonus).map {|l| l.summarize(false)},
-        stage_extras_level: {
-          kind: LEVEL_KIND.stage_extras,
-          icon: 'fa-flag',
-          title: "Stage Extras",
-          url: script_stage_stage_extras_url(script.name, stage_position: absolute_position),
-        },
+        stage_extras_level_url: script_stage_stage_extras_url(script.name, stage_position: absolute_position),
```

Comment:
By sending down a fake level object for stage extras, I was worried it would be easy at a future point to change the shape of Level.summarize, and not realize we needed to update this too. Given that the rest of this info is hardcoded, I decide to instead just pass the client a url, and let it own figuring out what to with it.